nextcloud/content.md

```diff
@@ -229,6 +257,16 @@ services:
 
 Then run `docker-compose up -d`, now you can access Nextcloud at http://localhost:8080/ from your host system.
 
+# Docker Secrets
+
+As an alternative to passing sensitive information via environment variables, _FILE may be appended to the previously listed environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in /run/secrets/<secret_name> files. For example:
```

Comment:
This functionality was reverted, wasn't it?

Reply:
Yes, I'll (re)implement this but I'd like to wait for https://github.com/docker-library/official-images/pull/5839.

Reply:
Removed this section for now.